Library/Formula/g3log.rb

```diff
@@ -0,0 +1,34 @@
+class G3log < Formula
+  desc "asynchronous, \"crash safe\", logger that is easy to use."
```

Comment:
So many possibilities... I went with the outer single quotes, so I can keep the double quotes in the text.